### ISSUE TITLE
NormalizeWhitespace improperly inserts a space before format specifiers in interpolated strings

### ISSUE THREAD

**Issue by tumtumtum** [dotnet/roslyn#24454]
**Version Used**: 

2.6.1

**Steps to Reproduce**:

1. Compile the following code

```cs
using System;
using Microsoft.CodeAnalysis;
using Microsoft.CodeAnalysis.CSharp;
using Microsoft.CodeAnalysis.CSharp.Syntax;

namespace ConsoleApp2
{
    class Program
    {
        static void Main(string[] args)
		{
			var code = 
			@"
				using System;

				public class Foo
				{
					public static void Main()
					{
						Console.WriteLine($""{Guid.NewGuid():N}"");
					}
				}
			";

			var node = SyntaxFactory.ParseSyntaxTree(code, new CSharpParseOptions(LanguageVersion.CSharp7_2))
				.GetCompilationUnitRoot()
				.NormalizeWhitespace("\t");

			Console.WriteLine(node);
		}
    }
}
```

**Expected Behavior**:

Should output:

```cs
using System;

public class Foo
{
	public static void Main()
	{
		Console.WriteLine($"{Guid.NewGuid():N}");
	}
}
```

**Actual Behavior**:

The output includes a space after the `:` and before the `N`. This causes the resulting program to fail to run because ": N" is not a valid format specifier.

```cs
using System;

public class Foo
{
	public static void Main()
	{
		Console.WriteLine($"{Guid.NewGuid(): N}");
	}
}
```

**Comment by sharwell** [comment]
Related to https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/2289#issuecomment-280200480

**Comment by tumtumtum** [comment]
Can you give me a pointer as to what it's changing in the syntax tree? I can't see anything different within the `FormatClauseSyntax` that would causing the space to be inserted.

**Comment by tumtumtum** [comment]
Oh typical, just after I write the message I find it. There's trailing whitespace trivia on the `ColonToken`

**Comment by jcouv** [comment]
Fix merged for 15.7
